packages/mobile/src/store/init/deepLink/deepLink.saga.test.ts

```diff
@@ -127,7 +139,38 @@ describe('deepLinkSaga', () => {
       .not.put(
         communities.actions.createNetwork({
           ownership: CommunityOwnership.User,
-          peers: validPairs,
+          peers: validData.pairs,
+          psk: validData.psk,
+        })
+      )
+      .run()
+  })
+
+  test('displays error if invitation code is invalid', async () => {
+    const invalidData: InvitationData = {
+      pairs: [
+        {
+          onionAddress: 'y7yczmugl2tekami7sbdz5pfaemvx7bahwthrdvcbzw5vex2crsr26qd',
+          peerId: 'QmZoiJNAvCffeEHBjk766nLuKVdkxkAT7wfFJDPPLsbKSE',
+        },
+      ],
+      psk: 'BNlxfE=',
+    }
+    const invalidCode = composeInvitationShareUrl(invalidData)
+    store.dispatch(
+      initActions.setWebsocketConnected({
+        dataPort: 5001,
+      })
+    )
+    const reducer = combineReducers(reducers)
+    await expectSaga(deepLinkSaga, initActions.deepLink(invalidCode))
+      .withReducer(reducer)
+      .withState(store.getState())
+      .not.put(
```

Comment:
You could additionally check if it puts `replaceScreen` but I think it's minor

Reply:
I had a problem with adding this because we pass arrow function as one of the params so it's not comparable. However I discovered that we can test `put` params partially with `put.like` so now it's done and works like a charm 👍🏻